src/PARIInterface.c

```diff
@@ -525,36 +530,27 @@ static Obj FuncPARI_CALL3(Obj self, Obj name, Obj a1, Obj a2, Obj a3)
     GEN g1 = PARI_DAT_GEN(a1);
```

Comment:
This is missing the 
```
GENFunc func = GetPARIFunc(name);
```
(and unfortunately the "suggestion" stuff doesn't work for this)

Reply:
Fixed. Yeah, some tests that exercise functions with 0 to 5 args would be useful :)